agent/containers/images/image-test-script.sh

```diff
@@ -0,0 +1,333 @@
+#!/bin/bash
+
+# A demonstration of how to use pbench-tool-meister-start with an existing
+# Redis server and one or more remote hosts
+export CONTROLLER=$(hostname -f)
+
+DISTRO_DEFAULT="centos-8"
+TAG_DEFAULT="a933ae45e"
+REDIS_HOST_DEFAULT=$CONTROLLER
+REDIS_PORT_DEFAULT=17001
+PBENCH_RUN_DEFAULT=/var/tmp/test-run-dir
+
+while :; do
+    read -p "Specify distribution to test (or press 'ENTER' for default ${DISTRO_DEFAULT}): "
+    DISTRO=${REPLY:-${DISTRO_DEFAULT}}
+    read -p "Specify image tag to test (or press 'ENTER' for default ${TAG_DEFAULT}): "
+    TAG=${REPLY:-${TAG_DEFAULT}}
+
+    printf -- "Pulling tool-data-sink image to test...\n"
+    if podman pull quay.io/pbench/pbench-agent-tool-data-sink-$DISTRO:$TAG ; then
+        printf -- "Pull complete!\n\n"
+        break
+    else
+        printf -- "Distro/tag combination could not be found, please try again.\n\n"
+    fi
+done
+
+while :; do
+    read -p "Specify redis host (or press 'ENTER' for default '$REDIS_HOST_DEFAULT'): "
+    REDIS_HOST=${REPLY:-${REDIS_HOST_DEFAULT}}
+    if ping -c 1 -W 1 ${REDIS_HOST} 2>&1 1>/dev/null ; then
+        printf -- "Host pinged and validated!\n\n"
+        break
+    else
+        printf -- "Host specified either does not exist or is unreachable, please try again:\n"
+    fi
+done
+
+while :; do
+    read -p "Specify Redis port (or press 'ENTER' for default port '${REDIS_PORT_DEFAULT}'): "
+    let REDIS_PORT=${REPLY:-${REDIS_PORT_DEFAULT}}
+    if ((1 <= $REDIS_PORT <= 65535)); then
+        break
+    else
+        printf -- "Port must be a non-zero numeric value; please retry\n"
```

Comment:
Well, technically, "must be a numeric value from 1 to 65535", which might be worth including here ... or someone could get errors where they're clearly following your instructions.